dev/AppAttach/AppAttachFramework/AVDAppAttachValidate/Scripts/AzureFileShareScript.ps1

```diff
@@ -0,0 +1,73 @@
+﻿param(
+        [Parameter(Mandatory=$false)]
+        [string] $accessToken,
+
+        [Parameter(Mandatory=$false)]
+        [string] $emailId,
+
+        [Parameter(Mandatory=$false)]
+        [string] $resourceGroupName,
+
+        [Parameter(Mandatory=$false)]
+        [string] $subscriptionId,
+
+        [Parameter(Mandatory=$false)]
+        [string] $storageAccountName,
+
+        [Parameter(Mandatory=$false)]
+        [string] $fileshareName,
+
+        [Parameter(Mandatory=$false)]
+        [string] $packageName,
+
+        [Parameter(Mandatory=$false)]
+        [string] $fileExistsInFileShare,
+
+        [Parameter(Mandatory=$false)]
+        [string] $fileDoesNotExistInFileShare
+    )
+
+try{
+    if(-not ($accessToken -and $emailId -and $resourceGroupName -and $subscriptionId -and $storageAccountName -and $fileshareName -and $packageName -and $fileExistsInFileShare -and $fileDoesNotExistInFileShare)){
+        throw "Mandatory parameters are missing in the script"
```

Comment:
Not the most helpful of error messages.

RECOMMEND: Provide more helpful error messages e.g.

```
if (-not $accessToken)
{
        throw "-AccessToken is required";
}
if (-not $resourceGroupName)
{
        throw "-ResourceGroupName is required";
}
...
```